velox/exec/tests/TaskTest.cpp

```diff
@@ -1018,7 +1018,7 @@ DEBUG_ONLY_TEST_F(TaskTest, liveStats) {
   EXPECT_EQ(32 * numBatches, operatorStats.outputBytes);
   EXPECT_EQ(3 * numBatches, operatorStats.outputPositions);
   EXPECT_EQ(numBatches, operatorStats.outputVectors);
-  EXPECT_EQ(1, operatorStats.finishTiming.count);
+  EXPECT_LE(1, operatorStats.finishTiming.count);
```

Comment:
This should be EXPECT_EQ as we don't collect backgroud time from exchange client yet? We might need a test Prestissimo which use local shuffle? Thanks!

Reply:
Done.